src/library/coverartdelegate.cpp

```diff
@@ -43,15 +43,14 @@ CoverArtDelegate::CoverArtDelegate(QTableView* parent)
 
 void CoverArtDelegate::emitRowsChanged(
         QList<int>&& rows) {
-    if (rows.isEmpty()) {
-        return;
-    }
     // Sort in ascending order...
     std::sort(rows.begin(), rows.end());
     // ...and then deduplicate...
     rows.erase(std::unique(rows.begin(), rows.end()), rows.end());
     // ...before emitting the signal.
-    DEBUG_ASSERT(!rows.isEmpty());
+    if (rows.isEmpty()) {
```

Comment:
Why is this changed? The commands in between are looking senseless in case of rows being empty. 
If they are there for a reason, please comment the source accordingly.

Reply:
You are right. Can't become empty when deduplicated. I'll revert and force push.